src/utils/constants/commandInfo.ts

```diff
@@ -0,0 +1,76 @@
+// Default have this for every built in command 
+export interface ICommandInformation {
+    name: string,
+    description: string,
+    contract: string
+}
+
+export function prettyPrintCommand (command: ICommandInformation) {
+    return `${command.name}\n${command.description}\ncontract: ${command.contract} \n`;
+}
+
+export const Ls_CommandInfo: ICommandInformation = {
+    name: "ls",
+    description: "List directory information",
+    contract: "ls"
+};
+
+export const Cd_CommandInfo: ICommandInformation = {
+    name: "cd",
+    description: "Enter directory",
+    contract: "cd <path>"
+};
+
+export const Pwd_CommandInfo: ICommandInformation = {
+    name: "pwd",
+    description: "Print current path",
+    contract: "pwd"
+};
+
+export const Code_CommandInfo: ICommandInformation = {
+    name: "code",
+    description: "Open file/folder/current directory",
+    contract: "code <path?>"
+};
+
+export const Clear_CommandInfo: ICommandInformation = {
+    name: "clear",
+    description: "Clear terminal",
+    contract: "clear"
+};
+
+export const Help_CommandInfo: ICommandInformation = {
+    name: "help",
+    description: "Get info about brsh",
+    contract: "help <command_name?>"
+};
+
+export const Install_CommandInfo: ICommandInformation = {
+    name: "install",
+    description: "Install a new wasi cmd to brsh",
+    contract: "install <wasi_url>"
```

Comment:
Currently it's `install <command name> <wasi_url>`  But maybe we should change this to `install <command name> <command description> <command usage> <wasi_url>`  The more scalable solution would be to expect a manifest file at the url location that has these details but we could start here if that's simpler.

Reply:
Ahhh that's a good point. I also think down the road I may probably convert this file into a manifest so it'll be easier to read. 

For now i think our current install command is good; I'm hoping if someone downloads a function they'll know what it does 🤞 . I added in a part to the help dialogue where we can also show users what wasi commands they have installed